tools/c7n_azure/c7n_azure/resources/activity_log.py

```diff
@@ -0,0 +1,47 @@
+from collections.abc import Iterator
+import datetime
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ArmResourceManager
+
+
+@resources.register('activity-log')
+class ActivityLog(ArmResourceManager):
+    """Activity Log Resource
+
```

Comment:
This feels a little strange, generally custodian models control plane resources and configurations. we don't generally expose data plane things as a resource. having individual audit api records as a resource feels like a mismatch.

What's the goal / use case for this?

 We do query data plane resources for specific functionality (filter/action) against a control plane resource, like auto tagging in the context of azure uses the activity log to determine creator.

Reply:
Our team have developed resources/filters for covering several security standards.
This resource covers one of the standards. If it's required we can create a set of resources/filters with corresponding rules in standards. Does it make sense?

Reply:
could you cite the standard here or explain the use case a bit more?  ie. as an analogy we typically ensure cloud trail or bucket configuration, not individual records or objects in those which could number in the billions.

Reply:
we use activity-log-alert for covering CIS standard (v7 (1.4,1.5), v8 (1.1)) - Each tag consists of a name and a value pair, which makes resource administration easier. We can create a table for these and other resource and in which standards we use the resources. How do you think, does it make sense?

Reply:
you'll have to be a little more specific re which cis standard or benchmark your referencing. v7 and v8 correspond to generic critical security controls. the latest azure specific benchmark is CIS Microsoft Azure
Foundations Benchmark v1.5.0 - 08-16-2022 none of the listed controls (1.1, 1.4, 1.5) correspond to functionality in the pr.

Reply:
also happy to jump on a zoom or hangout if it would be easier to talk through the details. I'm available on slack http://slack.cloudcustodian.io